From 73614e206a3ee822ef2ad1ac648a4616600ab87e Mon Sep 17 00:00:00 2001
From: Mike Gerdts <mike.gerdts@joyent.com>
Date: Tue, 24 Apr 2018 21:01:45 +0000
Subject: [PATCH] OS-6827 Update vmadm to capture new log after OS-6718
 Reviewed by: Josh Wilsdon <josh.wilsdon@joyent.com> Approved by: Josh Wilsdon
 <josh.wilsdon@joyent.com>

---
 src/vm/node_modules/VM.js | 21 ++++++++++-----------
 1 file changed, 10 insertions(+), 11 deletions(-)

diff --git a/src/vm/node_modules/VM.js b/src/vm/node_modules/VM.js
index c8c74997..d82ab6d8 100644
--- a/src/vm/node_modules/VM.js
+++ b/src/vm/node_modules/VM.js
@@ -6564,6 +6564,16 @@ function archiveVM(uuid, options, callback)
                 dst: archive_dirname,
                 targ: path.join(archive_dirname, 'cores')
             });
+            patterns_to_archive.push({
+                src: path.join(vmobj.zonepath, 'logs/console.log*'),
+                dst: path.join(archive_dirname, 'console'),
+                create_dst_dir: true
+            });
+            patterns_to_archive.push({
+                src: path.join(vmobj.zonepath, 'logs/platform.log*'),
+                dst: path.join(archive_dirname, 'platform'),
+                create_dst_dir: true
+            });
 
             if (vmobj.brand === 'kvm') {
                 patterns_to_archive.push({
@@ -6576,17 +6586,6 @@ function archiveVM(uuid, options, callback)
                     dst: archive_dirname,
                     targ: path.join(archive_dirname, 'startvm')
                 });
-            } else if (vmobj.brand === 'bhyve') {
-                patterns_to_archive.push({
-                    src: path.join(vmobj.zonepath, 'root/tmp/zhyve.log'),
-                    dst: path.join(archive_dirname, 'vmlogs'),
-                    create_dst_dir: true
-                });
-                patterns_to_archive.push({
-                    src: path.join(vmobj.zonepath, 'root/var/run/*.log*'),
-                    dst: path.join(archive_dirname, 'vmlogs'),
-                    create_dst_dir: true
-                });
             } else if (vmobj.docker) {
 
                 function logTimestamp() {
-- 
2.21.0

